linux_docker_resources/Dockerfile

```diff
@@ -110,9 +111,6 @@ RUN apt-get update && apt-get install --no-install-recommends -y libopencv-dev
 # Install console_bridge for class_loader et al.
 RUN apt-get update && apt-get install --no-install-recommends -y libconsole-bridge-dev
 
-# Install build dependencies for class_loader.
-RUN apt-get update && apt-get install --no-install-recommends -y libpoco-dev
-
```

Comment:
This should get split into a separate PR.

Reply:
All right, that's fine by me.  Just noticed it while doing this.  Will remove it here.

Reply:
Fixed in 3464ce1.